### ISSUE TITLE
Add cordova platform automatically.

### ISSUE THREAD

**Issue by mjsir911** [bhdouglass/clickable#53]
Hi,

I am building a cordova project from source code on git.
This means I haven't set it up at all, not initialized the `projects` directory or the `plugins` directory

When I type `clickable build`, I am given this error:
```
No clickable.json was found, using defaults and cli args
Auto detected template to be "cordova"
CordovaError: No platforms added to this project. Please use `cordova platform add <platform>`.
    at Object.preProcessOptions (/home/msirabella/.npm-packages/lib64/node_modules/cordova/node_modules/cordova-lib/src/cordova/util.js:196:15)
    at build (/home/msirabella/.npm-packages/lib64/node_modules/cordova/node_modules/cordova-lib/src/cordova/build.js:39:27)
    at cli (/home/msirabella/.npm-packages/lib64/node_modules/cordova/src/cli.js:222:26)
    at Object.<anonymous> (/home/msirabella/.npm-packages/lib64/node_modules/cordova/bin/cordova:41:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
Command '['cordova', '-d', 'build', 'ubuntu', '--device', '--', '--framework=ubuntu-sdk-15.04']' returned non-zero exit status 1.
```

Running `cordova platform add ubuntu` solves this problem and adds the `platforms` and `projects` directory, would it be trivial to add that into the build process?

**Comment by bhdouglass** [comment]
It seems trivial to add, but does it make sense to always add it? Although if you are using clickable that might be a safe assumption.

**Comment by mjsir911** [comment]
The `clickable build` command will fail every time if you do not have the (ubuntu?) platform. I think it's a fair assumption that if you want to build an app with the clickable command, you want to add the ubuntu platform to do so.

**Comment by mjsir911** [comment]
Fixed with #57
